src/controller/CommissioneeDeviceProxy.h

```diff
@@ -161,6 +164,13 @@ class CommissioneeDeviceProxy : public DeviceProxy, public SessionDelegate
     /* Compressed fabric ID and node ID assigned to the device. */
     PeerId mPeerId;
 
+    /*
+     * mPeerId can change when we get a NOC, but for purposes of stopping
+     * commissioning it's useful to allow clients to use the original ID they
+     * used as a commissioning process identifier.
+     */
+    NodeId mOriginalDeviceId;
```

Comment:
Maybe calling this the "Temporary Commissioning ID" or something like that would be clearer than "Original ID" ?